public/app/partials/reset_password.html

```diff
@@ -10,7 +10,7 @@ <h3 class="p-b-1">Reset password</h3>
 		<div ng-if="disableLoginForm">
 			You cannot reset password when login form is disabled.
 		</div>
-		<form name="sendResetForm" class="login-form gf-form-group" ng-show="mode === 'send'" ng-hide="ldapEnabled || authProxyEnabled || disableLoginForm">
+		<form name="sendResetForm" class="login-form gf-form-group" ng-show="mode === 'send'" ng-hide="ldapEnabled || authProxyEnabled || disableLoginForm || mode === 'reset'">
 			<div class="gf-form">
 					<span class="gf-form-label width-7">User</span>
```

Comment:
Hm, so when is the Username form used at all?

Reply:
When you click "Forgot password" on login page.